just-install.json

```diff
@@ -1575,9 +1582,9 @@
         "options": {
           "destination": "{{.PROGRAMFILES_X86}}\\Rufus\\rufus.exe"
         },
-        "x86": "https://rufus.akeo.ie/downloads/rufus-{{.version}}.exe"
+        "x86": "https://rufus.akeo.ie/downloads/rufus.exe"
       },
-      "version": "3.1"
+      "version": "latest"
```

Comment:
Is there a reason for this change?

Reply:
I made this change for future update so we don't have to update it manually with the latest entry.
In addition, it is recommended to always use "lastest" when possible in the Readme: "refer unversioned entries: if a software provides an URL to always get the latest installer try to use that."

Reply:
@eveldee0680 OK, sounds good.